sync/api/projects.py

```diff
@@ -54,6 +55,8 @@ def create_project(
     :type s3_url: str, optional
     :param prediction_preference: preferred prediction solution, defaults to Preference.BALANCED
     :type prediction_preference: Preference, optional
+    :param prediction_params: dictionary of prediction parameters, defaults to None. Valid options are documented here - https://developers.synccomputing.com/reference/create_project_v1_projects_post
```

Comment:
Seemed better to not try to duplicate API docs in this comment, but I also think this API documentation is getting replaced at some point?.. Or at least, the url may end up changing.